en/docs/develop-components/bring-your-own-image.md

```diff
@@ -1,79 +1,76 @@
 # Bring Your Own Image (BYOI)
 
-Deploy and manage (prebuilt) container images from external container registries as Choreo Components. This enables you to deploy and effectively manage your container images directly within the Choreo environment.
+Choreo allows you to deploy and manage prebuilt container images from external container registries as Choreo components. This enables you to deploy and effectively manage your container images within the Choreo environment.
 
 !!! info
-    This feature is currently only available on [Private Data Planes (PDP)](https://wso2.com/choreo/docs/choreo-concepts/data-planes/#private-data-plane) and supports the following component types:
+    This feature is currently only available on [private data planes (PDPs)](../choreo-concepts/data-planes.md#private-data-planes) for the following component types:
 
-    - service
-    - web applications
-    - scheduled tasks
-    - manual tasks.
+    - Service
+    - Web Application
+    - Scheduled Task
+    - Manual Task
 
 
 ## Prerequisites
 
-Before you try out this guide, be sure you have the following:
+Before you try out this guide, ensure you have the following:
 
-- **A container registry**:  A container registry that contains the images you want to deploy. 
+- **A container registry**: Ensure you have a container registry containing the images you want to deploy. Choreo is compatible with various container registries, including but not limited to GCR (Google Container Registry), ACR (Azure Container Registry), GitHub Container Registry, and Docker Hub.
```

Comment:
Shall we add Amazon Elastic Container Registry  instead of GitHub Container Registry?

Reply:
+1 
There is a new task [1] to test the BYOI steps and update to align with latest UIs and other necessary changes. I'll add this suggestion to that.
[1] https://github.com/wso2-enterprise/choreo/issues/29664